packages/video_player/tizen/src/video_player.cc

```diff
@@ -78,6 +80,35 @@ FlutterDesktopGpuSurfaceDescriptor *VideoPlayer::ObtainGpuSurface(
   return gpu_surface_.get();
 }
 
+void VideoPlayer::InitScreenSaverApi() {
+  screensaver_handle_ = dlopen("libcapi-screensaver.so", RTLD_LAZY);
+  if (!screensaver_handle_) {
+    LOG_INFO("[VideoPlayer] dlopen failed: %s", dlerror());
```

Comment:
`LOG_ERROR` seems more appropriate than `LOG_INFO` (same for below).

Reply:
Fixed.